plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/ReadSessionCreator.java

```diff
@@ -129,76 +109,12 @@ private TableInfo getActualTable(
             // get it from the view
             String query = bigQueryClient.selectSql(table.getTableId(), requiredColumns);
```

Comment:
I would move this into `bigQueryClient.getCachedTable` as well

Reply:
Done.